### PR TITLE
Use string types compatible with py2 and py3

### PR DESCRIPTION
**Pull request by puiterwijk**
This enables Python3 support.
It is based on top of the CI branch to add CI testing for python3 at the same time.

### CODE REVIEW

**Review by kashyapc — COMMENTED** [review]
(Edit: Ah, I should have written this comment on: https://github.com/puiterwijk/qemu-ovmf-secureboot/pull/3/commits/1dceba6 )

As discussed on IRC, perhaps let's make this an argument? So that if I want to run  it on my laptop, I can use hardware acceleration; and in build environments, can use the pure emulation. :-)

File: ovmf-vars-generator [hunk 1]
```diff
@@ -136,9 +140,10 @@ def test_keys(args):
             read = p.stdout.readline()
             if args.print_output:
                 print('OUT: %s' % strip_special(read), end='')
-            if 'Secure boot disabled' in read:
+                print()
+            if b'Secure boot disabled' in read:
```
Comment (kashyapc) [inline ovmf-vars-generator:144]: Ultra nit: s/Secure boot/Secure Boot/g/

Comment (puiterwijk) [reply ovmf-vars-generator:144]: This is the output from the Linux kernel, so we can't really change this.

Comment (kashyapc) [reply ovmf-vars-generator:144]: Hehe, indeed.  I did think of it for a moment, and then realized it after hitting submit on the comment. Disregard it, then.

**Review by kashyapc — APPROVED** [review]
Looks good to me.

**Pull request merged**